index.html

```diff
@@ -0,0 +1,173 @@
+             </div> -->
+        </div>
+      
+
+                 <div class="program-wrapper">
+                  <div class="title-wrap">
+                  <h3 class="pro-header">Main Program</h3>
+                  <hr>
+                 </div>
+        
+                      <div class="main-program">
+          
+                        <div class="program-card">
+                          <img src="images/speech.png" alt="" class="pro-icon">
+                          <h4 class="program">Lecture</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/inov2.png" alt="Innovators Exhibition" class="pro-icon">
+                          <h4 class="program">Inovators</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/invest.png" alt="Investments Forum" class="pro-icon">
+                          <h4 class="program">Investers</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/net.jpg" alt="Networking" class="pro-icon">
+                          <h4 class="program">Networks</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                       </div>
+
+                              <div class="advert">
+                                <p>Join WFH Dev's Annual global  2022</p>
+                              </div>
+                              <div class="advert1">
+                                <p>SEE THE WHOLE PROGRAM</p>
+                              </div>
+                 </div>
+        <div class="speaker-head">
+        <h3 class="speaker-header">Featured Speakers</h3>
+        <hr>
+      </div>
+        <div class="speakers"></div>
+        
+
+        <div class="partners-sect">
+          <div class="partner-header"><p>Partners</p> <hr></div>
+          <div class="partners">
+              <img src="images/google-1-1.svg" alt="">
+              <img src="images/meta-1.svg" alt="">
+              <img src="images/microsoft.svg" alt="">
+              <img src="images/coinbase.svg" alt="">
+              <img src="images/mozilla-1.svg" alt="">
+
+          </div>
+
+       </div>
```

Comment:
- [ ] In the partners section, the "COINBASE" logo takes more space as the width is increased.
**Let's fix it so that all the logos take up equal space and scale equally  so that it appears more balanced. Cheers** 👍 

![image](https://user-images.githubusercontent.com/60618877/183242900-59ef87fa-c03c-4c26-8ed9-417c12fae682.png)